core/configuration/ignored_identifier.py

```diff
@@ -87,7 +87,7 @@ def __init__(self, *args, **kargs) -> None:
 
     def _get_ignored_identifier_types(
         self, configuration: IgnoredIdentifierConfiguration
-    ) -> Set[int]:
+    ) -> Any:
```

Comment:
Same question about less granular types

Reply:
Pretty much the same as above since this is calling that method already mentioned.